test/api/profileTypes.test.js

```diff
@@ -1,33 +1,318 @@
 import 'test/enzyme-init';
-import { getProfileTypes } from 'api/profileTypes';
 import {
+  postProfileType,
+  putProfileType,
+  deleteProfileType,
+  getProfileType,
+  getProfileTypes,
+  getAttributeFromProfileType,
+  deleteAttributeFromProfileType,
+  postAttributeFromProfileType,
+  putAttributeFromProfileType,
+  getProfileTypeAttributes,
+  getProfileTypeAttribute,
+} from 'api/profileTypes';
+
+import { makeRequest, METHODS } from '@entando/apimanager';
+
+import {
+  PROFILE_TYPES,
+  PROFILE_TYPES_DELETE_OK,
+  ATTRIBUTE_PROFILE_TYPES_DELETE_OK,
   PROFILE_TYPES_OK_PAGE_1,
-  PROFILE_TYPES_OK_PAGE_2,
+  PROFILE_TYPES_ATTRIBUTES,
+  PROFILE_TYPE_ATTRIBUTE,
 } from 'test/mocks/profileTypes';
 
+const correctRequest = {
+  uri: '/api/profileTypes',
+  method: METHODS.GET,
+  mockResponse: PROFILE_TYPES_OK_PAGE_1,
+  useAuthentication: true,
+};
+
+
 jest.unmock('api/profileTypes');
-jest.useFakeTimers();
+jest.mock('@entando/apimanager', () => ({
+  makeRequest: jest.fn(() => new Promise(resolve => resolve({}))),
+  METHODS: {
+    GET: 'GET', POST: 'POST', PUT: 'PUT', DELETE: 'DELETE',
+  },
+}));
+
+describe('api/postProfileType', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('returns a promise', () => {
+    expect(postProfileType()).toBeInstanceOf(Promise);
+  });
+
+  it('if successful, returns a mock ok response', () => {
+    postProfileType(PROFILE_TYPES);
+    expect(makeRequest).toHaveBeenCalledWith({
+      ...correctRequest,
```

Comment:
What's the point in copying the whole GET request and then overriding almost all of its properties? It would be much more readable, coherent and effective if you just wrote the request object from scratch.